src/main/steps/applicant1/occupation/content.test.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable jest/expect-expect */
+import { FormContent, FormFields } from '../../../app/form/Form';
+import { CommonContent, generatePageContent } from '../../common/common.content';
+
+import { cy as cyFunction, en as enFunction, generateContent } from './content';
+
+const CY = 'cy';
+const EN = 'en';
+const cyContent = cyFunction();
+const enContent = enFunction();
+
+const langAssertions = (language, content) => {
+  const generatedContent = generateContent({ language, userCase: {} } as CommonContent);
+  const { section, title, occupation, warningText, details, errors } = content;
+
+  expect(generatedContent.section).toEqual(section);
+  expect(generatedContent.title).toEqual(title);
+  expect(generatedContent.occupation).toEqual(occupation);
+  expect(generatedContent.warningText).toEqual(warningText);
+  expect(generatedContent.details).toEqual(details);
+  expect(generatedContent.errors).toEqual(errors);
+};
+
+const commonContent = { language: EN } as CommonContent;
+
+describe('occupation content', () => {
+  it('should return the correct content for language = en', () => {
+    langAssertions(EN, enContent);
+  });
+
+  it('should return the correct content for language = cy', () => {
+    langAssertions(CY, cyContent);
+  });
+
+  it('should have an occupation text input field', () => {
+    const generatedContent = generateContent(commonContent);
+    const form = generatedContent.form as FormContent;
+    const fields = form.fields as FormFields;
+    const applicant1Occupation = fields.applicant1Occupation;
+
+    expect(applicant1Occupation.type).toBe('input');
+    expect((applicant1Occupation.label as Function)(generateContent(commonContent))).toBe(enContent.occupation);
+    expect(applicant1Occupation.labelSize).toBe('normal');
+    expect(applicant1Occupation.classes).toBe('govuk-input--width-20');
```

Comment:
test missing for `validator`

Reply:
Had some issues with this when it has multiple validators, let's connect and discuss